custom/enikshay/reports/repeaters.py

```diff
@@ -46,6 +46,8 @@ class ENikshayForwarderReport(DomainForwardingRepeatRecords):
     def get_all_rows(self):
         repeater_id = self.request.GET.get('repeater', None)
         state = self.request.GET.get('record_state', None)
+        if self.is_rendered_as_email:
+            return []
```

Comment:
It would be better to not even call this method if it is to return []. Or atleast return at first step instead of fetching repeater and state first?